_config/protected.yml

```diff
@@ -10,3 +10,6 @@
 - /video/youtube/i1aiUaa8RZ8/
 - /video/youtube/LbzkY9nrgdo/
 - /video/youtube/Q37niG_hCcA/
+- /video/youtube/pIURFD15WJ0/
+- https://plus.google.com/events/cmoaedv5ib92rlhh9bnjrkd1hhg?authkey=CNnNnODpxtTzbQ
+- /webinars/net-on-rhel-sneak-peek
```

Comment:
FYI: the reg-walling works for other pages in this PR: http://developers-pr.stage.redhat.com/pr/956/build/655//video/youtube/I-Po2phF-_M